pkg/controller/rds/dbinstance/setup.go

```diff
@@ -153,6 +153,10 @@ func (e *custom) preUpdate(ctx context.Context, cr *svcapitypes.DBInstance, obj
 	if pwchanged {
 		obj.MasterUserPassword = aws.String(pw)
 	}
+
+	if aws.BoolValue(cr.Spec.ForProvider.ApplyImmediately) {
+		obj.ApplyImmediately = cr.Spec.ForProvider.ApplyImmediately
+	}
```

Comment:
```suggestion
		obj.ApplyImmediately = cr.Spec.ForProvider.ApplyImmediately
```
Does that make a difference?